spec/unit/ConfigurationTest.rb

```diff
@@ -22,9 +22,9 @@ def test_new_nonempty_string
 
   def test_api_base
     assert_equal 'http://api.local.dev:3000', PaymentRails::Configuration.new('key', 'secret', 'integration').apiBase
-    assert_equal 'https://api.paymentrails.com', PaymentRails::Configuration.new('key', 'secret', 'production').apiBase
+    assert_equal 'https://api.trolley.com', PaymentRails::Configuration.new('key', 'secret', 'production').apiBase
     assert_equal 'https://api.railz.io', PaymentRails::Configuration.new('key', 'secret', 'development').apiBase
```

Comment:
Should we remove this?

Reply:
There's a task in 'Gap updates' to remove the environments, this can be handled in that task